cloudcompose/cluster/aws/cloudcontroller.py

```diff
@@ -139,7 +139,8 @@ def cleanup(self):
     def _resolve_ami_name(self, upgrade_image):
         if self.aws['ami'].startswith('ami-'):
             return self.aws['ami']
-
+        # Set the ami name so we can reference it downstream
+        self.aws['ami_name'] = self.aws['ami']
```

Comment:
Oh sorry I meant to type out a description and forgot.  So, on `cloud-compose cluster up` that variable is overwritten with the actual ami name so templates like this https://github.com/washingtonpost/docker-ecs/blob/master/cloud-compose/templates/docker.config.sh#L4 don't work